curriculum/challenges/english/10-coding-interview-prep/the-odin-project/introduction-to-html-css-question-b.md

```diff
@@ -0,0 +1,37 @@
+---
+id: 6376327e2724a688c04636e3
+videoId: LGQuIIv2RVA
+title: Introduction To HTML and CSS Question B
+challengeType: 11
+dashedName: introduction-to-html-and-css-question-b
+---
+
+# --description--
+
+
+HTML and CSS are two languages that work together to create everything that you see when you look at something on the internet. HTML is the raw data that a webpage is built out of. All the text, links, cards, lists, and buttons are created in HTML. CSS is what adds style to those plain elements. HTML puts information on a webpage, and CSS positions that information, gives it color, changes the font, and makes it look great!
+
+Many helpful resources out there keep referring to HTML and CSS as programming languages, but if you want to get technical, labeling them as such is not quite accurate. This is because they are only concerned with presenting information. They are not used to program logic. JavaScript, which you will learn in the next section, is a programming language because it’s used to make webpages do things. Yet, there is quite a lot you can do with just HTML and CSS, and you will definitely need them both. Throughout our curriculum, the following lessons focus on giving you the tools you need to succeed once you reach JavaScript content.
```

Comment:
We repeat this information over four different lessons - I don't think that's right.